frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -79,3 +84,5 @@ frontend/packages/dev-console/integration-tests/
 1. Don't use static sleep statements (browser.sleep)
 2. Comments should be included wherever required
 3. Don't inlcude console.log statements while raising PR
+4. .gherkin-lintrc configuration file present in frontend folder is used to set Gherkin standards
+5. Execute the "npm run gherkin-lint" command for every QE pr
```

Comment:
We use `yarn` throughout the READMEs, recommend you don't refer to `npm` (despite it being a dependency of `yarn`.

Reply:
updated